packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/ListView/tests/index.test.js

```diff
@@ -72,7 +72,13 @@ describe('ADMIN | Pages | API TOKENS | ListPage', () => {
 
   it('should show a list of api tokens', async () => {
     useRBAC.mockImplementation(() => ({
-      allowedActions: { canCreate: true, canDelete: true, canRead: true, canUpdate: true },
+      allowedActions: {
+        canCreate: true,
+        canDelete: true,
+        canRead: true,
+        canUpdate: true,
+        canRegenerate: true,
```

Comment:
just wondering: I don't see `canRegenerate` data used in the ListView, do we need to mock it for the tests? 🤔

Reply:
ehy, we don't need the canRegenerate in the list view because we show the regenerate button only in the Edit view

Reply:
yep that's what I understood, but I was wondering why you needed to mock `canRegenerate` in the `ListView` tests since it's not used there? 🙂